include/cantera/zeroD/WallClasses.h

```diff
@@ -0,0 +1,11 @@
+//! @file WallClasses.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_WALL_CLASSES_H
+#define CT_WALL_CLASSES_H
+
+#include "cantera/zeroD/Wall.h"
+
+#endif
```

Comment:
Again, I would prefer to just leave wall implementations as a includes directly in `zerodim.h`.

Reply:
These `.h` files made more sense in the context of the `ic_engine` implementation. I don't have a strong opinion.

Reply:
removed.